src/shared/services/configuration/map-markers.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable global-require */
+import L from 'leaflet';
+
+export const smallMarkerIcon = global.window.L.icon({
```

Comment:
You're using `global.window.L.icon` here and `L.icon` on line 10. Won't that conflict? I'd use one notation and stick with that.

Reply:
You are right, I have corrected that.